tests/components/devolo_home_control/test_config_flow.py

```diff
@@ -15,28 +17,28 @@
 from tests.common import MockConfigEntry
 
 
-async def test_form(hass):
+async def test_form(hass: HomeAssistant):
```

Comment:
I think you could add the return type hint to all the test definition here.
```suggestion
async def test_form(hass: HomeAssistant) -> None:
```

Reply:
I added them +1